File: Examples/Ruby/Slides/get_slides_theme_color_scheme.rb
#ExStart:
require 'aspose_slides_cloud'

class Slides

  include AsposeSlidesCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("", "")
    @slides_api = SlidesApi.new
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../../../data/" << file_name,"r") { |io| io.read } )
  end

  # Read slide theme color scheme info.
  def get_slides_theme_color_scheme
  	file_name = "sample.pptx"
  	upload_file(file_name)
  
  	slide_index = 1
  	response = @slides_api.get_slides_theme_color_scheme(file_name, slide_index)
  end

end

slides = Slides.new()
puts slides.get_slides_theme_color_scheme
#ExEnd:

File: Examples/PHP/Slides/GetSlidesThemeColorScheme.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Slides\SlidesApi;
use Aspose\Slides\AsposeApp;

class Slides {

	public $slidesApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->slidesApi = new SlidesApi();
	}

	public function getSlidesThemeColorScheme() {
		$fileName = "sample.pptx";
		Utils::uploadFile($fileName);
		
		$result = $this->slidesApi->GetSlidesThemeColorScheme($fileName, $slideIndex = 1, $folder = "", $storage = "");
		print_r($result);
	}
}

$slides = new Slides();
$slides->getSlidesThemeColorScheme();

?>
//ExEnd:
